test/e2e/functional_test.go

```diff
@@ -0,0 +1,368 @@
+package prometheus
```

Comment:
You are missing the copyright header here, see -> https://github.com/openshift/cluster-monitoring-operator/blob/master/test/e2e/telemeter_test.go#L1-L13